cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_annotate.py

```diff
@@ -0,0 +1,57 @@
+'''
+Created on Aug. 12, 2024
+
+@author: pleyte
+'''
+
+import logging
+import os
+
+from biocommons.seqrepo import SeqRepo
+
+from edu.ohsu.compbio.txeff.util import chromosome_map
+from edu.ohsu.compbio.txeff.variant_transcript import VariantTranscript
+
+
+class TxEffAnnotate(object):
+    '''
+    Add variant annotations to each variant transcript.  
+    '''
+    def __init__(self):
+        '''
+        Constructor
+        '''
+        # Set up a logger for this class
+        self.logger = logging.getLogger(__name__)
+        
+        # Setup the SeqRepo utility that will be used for the Reference Context annotation
+        if os.environ.get('HGVS_SEQREPO_DIR') is None:
+            raise ValueError("The HGVS_SEQREPO_DIR environment variable is not defined")
```

Comment:
You don't have to have SeqRepo installed locally. If you instantiate SeqRepo() without a parameter it will use their own (remote) repository. But they block requests from OHSU IP addresses, and we don't want to spend time making remote requests anyway, so an error is thrown if a local SeqRepo isn't installed.

Reply:
Sometimes I wonder how institutions like this came to blocklist OHSU but I'm grateful they do.